atomic_reactor/plugins/post_compare_components.py

```diff
@@ -91,6 +94,10 @@ def run(self):
                 t = component['type']
                 name = component['name']
 
+                if name in package_comparison_exceptions:
+                    self.log.warn("Ignoring comparison of package %s", name)
```

Comment:
Wouldn't info be more appropriate? atomic reactor is ignoring the package that reactor config told it to ignore. Logging it is a good idea, but it's not a potential bug.

Reply:
Yes, you're right. Updated.